### PR TITLE
PlayLed feature

### PR DESCRIPTION
**Pull request by qharley**
added back slow flicker function to indicate system suspend status

### CODE REVIEW

File: src/modules/utils/PlayLed/PlayLed.cpp [hunk 1]
```diff
@@ -54,8 +57,25 @@ uint32_t PlayLed::led_tick(uint32_t)
 
     if(++cnt >= 6) { // 6 ticks ~ 500ms
         cnt= 0;
-        led.set(!THEKERNEL->conveyor->is_queue_empty());
```
Comment (qharley) [inline src/modules/utils/PlayLed/PlayLed.cpp]: will add back...

**Comment by wolfmanjm** [comment]
Sorry but suspend is not a system state like pause was so this is not appropriate IMO.

**Comment by arthurwolf** [comment]
@wolfmanjm maybe we could have an optional output pin that would turn on when suspend is issued and off when resume is issued ?

**Comment by wolfmanjm** [comment]
I'd reconsider if this was optional, and defaults to off. and if it did not use public request to get the suspend status. basically that means moving Player to Kernel, and querying it directly.
I still do not consider suspend a system status though, just as pause from the host (or for sdcard) is not a system status. and playled is supposed to indicate system status IMO.

**Comment by qharley** [comment]
Interesting...

No problem.  At least the Morgans will keep behaving the same way even
though the pause function has been removed.

I used code borrowed from the panel module (public request).  Thought it
would be acceptable since it is used already...

**Comment by wolfmanjm** [comment]
Well pause was removed becuase it was totally broken. should never have been used. We had been talking about that for a while, it was actually deprecated over a year ago.

As for suspend, it is a totally different thing in my view. Halt (and pause) were system states and nothing could be done until they were out of that state. suspend is not a system state just as pausing a print from pronterface or stopping a print. there is no change in the system state and the print can be restarted or not and it can be resumed or not, the system itself is not stopped, just the current print is suspended.

**Comment by qharley** [comment]
Ok.  I suppose it has a lot to do with perception. Even though the state in
the software did not change, the machine is waiting for the user, and I
want to indicate that it is.

No problem.  I will maintain it in my own branch.

**Pull request closed without merging**